app/views/ships/index.html.erb

```diff
@@ -1 +1,62 @@
-<h1> SHIPS </h1>
\ No newline at end of file
+<h1 class="text-center"> Place Your Ships! </h1>
+<p class="text-center" id="inst">Click and drag the ships to the grid. <br/>
+   Click on ship to position it horizontally or vertically.</p>
+
+
+
+<div id="wrapper">
+    <% (0..10).each do |row| %>
+      <% if row > 0 %>
+        <div class="square textLabel"><%= row %></div>
+      <% else %>
+        <div class="square"></div>
+      <% end %>
+      <% (1..10).each do |col| %>
+        <% if row == 0 %>
+          <div class="square textLabel"><%= col %></div>  
+        <% else %>  
+          <div class="square" dataCoords="<%= col %><%= row %>">

```

Comment:
I might suggest using `data-column="<%= col %>" data-row="<%= row %>"` to make it easier to fetch.